ovos_PHAL_plugin_commonIOT/device_manager.py

```diff
@@ -35,6 +35,79 @@ def __init__(self, bus):
         # self.bus.on("ovos.iot.device.mute", self.handle_mute)
         # self.bus.on("ovos.iot.device.unmute", self.handle_unmute)
 
+        # iot lights actions
+        self.bus.on("ovos.iot.device.get.brightness", self.handle_get_brightness)
+        self.bus.on("ovos.iot.device.set.brightness", self.handle_set_brightness)
+        self.bus.on("ovos.iot.device.decrease.brightness", self.handle_decrease_brightness)
+        self.bus.on("ovos.iot.device.increase.brightness", self.handle_increase_brightness)
+        self.bus.on("ovos.iot.device.get.color", self.handle_get_color)
+        self.bus.on("ovos.iot.device.set.color", self.handle_set_color)
+
+    # light handlers
+    def handle_get_brightness(self, message):
+        device = self.search_for_device(message.data.get("device"))
+        if device:
+            return device.brightness
```

Comment:
should emit a bus message with response, this is a event so the return does nothing

same for other handlers in this file